src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -209,7 +209,7 @@ public ManagementCommandAck(bool status)
 
         #region Messages internal to EndpointManager
 
-        public sealed class Prune : INoSerializationVerificationNeeded { }
+        public sealed class Prune : INoSerializationVerificationNeeded, IAutoReceivedMessage { }
```

Comment:
`IAutoReceivedMessage ` already also implements: `INoSerializationVerificationNeeded`.
And again it seems rather random, to mark this message, but leave the others untouched.